mockupdb/__init__.py

```diff
@@ -275,7 +276,10 @@ def wait_until(predicate, success_description, timeout=10):
 
 OP_MSG_FLAGS = OrderedDict([
     ('checksumPresent', 1),
-    ('moreToCome', 2)])
+    ('moreToCome', 2),
+    ('exhaustAllowed', 16)])
```

Comment:
Unrelated, adding the exhaustAllowed flag.

Reply:
Gotcha.